lti_consumer/lti_xblock.py

```diff
@@ -100,11 +100,11 @@
     'instructor': 'Instructor',
 }
 CUSTOM_PARAMETER_SEPARATOR = '='
-CUSTOM_PARAMETER_TEMPLATE_PATTERN = r'\${ *\w+ *}'
-CUSTOM_PARAMETER_TEMPLATE_REGEX = re.compile(CUSTOM_PARAMETER_TEMPLATE_PATTERN)
+CUSTOM_PARAMETER_CHARS_SET = r'[\w!#$%&()*+,\-.:;<>?@\[\]^{|}~]'
```

Comment:
Could you please take a look at this? Maybe we can reuse something. @kuipumu 
```
>>> import string
>>> string.punctuation
>>> string.printable
>>> string.whitespace
```

Reply:
@Jacatove string.punctuation Is a similar set but there are various characters that need to be removed in order to work with the XBlock field (Ex: "'`\) and also need to be escaped so they can be used inside the regex character class (Ex: []-). I could modify string.punctuation but I thing is better to be have the set defined ourselves.